### ISSUE TITLE
Where to put design assets

### ISSUE THREAD

**Issue by ellisonbg** [jupyter/design#33]
In the past, we have been putting design assets on both:

1. A shared DropBox folder
2. This repo

The reason is that many of our designers haven't been git literate, and trying to diff/merge Illustrator files is really dangerous. The files in these two locations have fallen out of sync (they are completely different now) and the files are sort of a mess.

There are also a lot of "working" files, such as Illustrator and Sketch files that the designers use.

We need to organize and consolidate these assets.

Questions:

* Where to put all of our final design files?
* Where to put more of the working design files?
* Should we force everyone to use Git?

@jupyter/designers

**Comment by fperez** [comment]
I really would like to move, as much as possible, to a public workflow.  I understand the challenges with using github for certain types of assets, and we can keep looking for a better solution. But what worries me about Dropbox (and similar) is that this part of the project becomes effectively near-private development: it becomes hard to contribute directly, since (even if we post screenshots and discussions on GitHub) only some people have access to the actual assets and full development process.

Not that I have a magic solution under my hat, mind you... But I really think we need to prioritize finding a solution that is as open and inclusive as the rest of the project: the whole "optimize for your open community" mantra we've heard already from folks who have fought these battles...

**Comment by spoorthyv** [comment]
We could use github and then have extremely newbie friendly instructions in the read-me on how to download the assets for designers that dont have github experience.

**Comment by spoorthyv** [comment]
I also think it would be nice to have a page on the website itself with all the brand guidelines and assets.

**Comment by fperez** [comment]
Certainly, that is slowly but surely being worked on too, and we'll consolidate all usage guidelines for name, brand, logos, etc. in one place.

**Comment by cameronoelsen** [comment]
So did we ever find a consensus on what the best way of doing this is?

**Comment by spoorthyv** [comment]
I think my tentative plan was to put everything in the dropbox and use that for daily use. Then also put the main set of logos and brand guidelines in the github.

**Comment by Ruv7** [comment]
@spoorthyv @faricacarroll @cameronoelsen @ellisonbg @fperez - let's move forward with putting a main set of logos in GitHub. We can use Dropbox for all assets and leave GitHub for the most widely used. I'm writing the headline story for the next newsletter on our new brand guidelines. This is set to go out Wednesday morning. Farica has already uploaded the existing brand guidelines (though we will continue iterating on this doc) if we can also get a set of logos up by tomorrow morning that will allow me to guide the community on where to find these items.

**Comment by cameronoelsen** [comment]
@Ruv7 I'm on it!

**Comment by faricacarroll** [comment]
The logos are accessible through dropbox and a logos folder on GitHub (thank you Spoo!). I will be uploading a single page document to explain when it is appropriate to use which logo. We will also be renaming condensed logo to either vertical or square logo. 

Planning on closing this issue within the next two weeks.

**Comment by Ruv7** [comment]
Great, thanks - please also post links when you close the issue. To clarify for anyone reading the single page document will be a legend of the logos created to guide users on the various logo files that will be available for download.
